packages/components/src/DocumentUpload/SecurityBlurb.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Icon } from "@jobber/components";
+import styles from "./DocumentUpload.module.css";
+import { SecurityBlurbProps } from "./types";
+
+export const SecurityBlurb: React.FC<SecurityBlurbProps> = ({
+  privacyPolicyUrl,
+}) => (
+  <div className={styles.securityBlurb}>
+    <Icon name="lock" size="small" className={styles.securityIcon} />
+    <div className={styles.securityText}>
+      Documents and files you upload are stored securely as per our{" "}
```

Comment:
a couple things here

- first it's putting text directly in a div rather than some kind of text element which I don't love
- second, the `{" "}` is an interesting way to add a space. I get that a formatter might remove a trailing whitespace but still. I don't love it.